src/spec.rs

```diff
@@ -35,9 +36,12 @@ impl ScrollSpecId {
     /// Converts the `ScrollSpecId` to a `SpecId`.
     const fn into_eth_spec_id(self) -> SpecId {
         match self {
-            Self::SHANGHAI | Self::BERNOULLI | Self::CURIE | Self::DARWIN | Self::EUCLID => {
-                SpecId::SHANGHAI
-            }
+            Self::SHANGHAI
+            | Self::BERNOULLI
+            | Self::CURIE
+            | Self::DARWIN
+            | Self::EUCLID
+            | Self::FEYNMAN => SpecId::SHANGHAI,
```

Comment:
Should we use Shanghai for all Scroll spec variants?

Reply:
I think so, this is also done by the zk team, I don't think our spec includes all the changes from Cancun.